### PR TITLE
Fixed a typo - 'Glimmer components have a some huge benefits' to ...

### PR DESCRIPTION
**Pull request by Mithrilhall**
Fixed a typo - 'Glimmer components have a some huge benefits' to 'Glimmer components have some huge benefits'.

### CODE REVIEW

File: guides/release/upgrading/current-edition/glimmer-components.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Ember, and you don't need to worry about learning it separately.
 
 ## Benefits of Glimmer Components
 
-Glimmer components have a some huge benefits
+Glimmer components have some huge benefits
```
Comment (amyrlam) [inline guides/release/upgrading/current-edition/glimmer-components.md:25]:
```suggestion
Glimmer components have some huge benefits:
```

File: guides/v3.15.0/upgrading/current-edition/glimmer-components.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Ember, and you don't need to worry about learning it separately.
 
 ## Benefits of Glimmer Components
 
-Glimmer components have a some huge benefits
+Glimmer components have some huge benefits
```
Comment (amyrlam) [inline guides/v3.15.0/upgrading/current-edition/glimmer-components.md:25]:
```suggestion
Glimmer components have some huge benefits:
```

File: guides/v3.16.0/upgrading/current-edition/glimmer-components.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Ember, and you don't need to worry about learning it separately.
 
 ## Benefits of Glimmer Components
 
-Glimmer components have a some huge benefits
+Glimmer components have some huge benefits
```
Comment (amyrlam) [inline guides/v3.16.0/upgrading/current-edition/glimmer-components.md:25]:
```suggestion
Glimmer components have some huge benefits:
```

**Comment by amyrlam** [comment]
@jenweber @locks not sure why the CI didn't kick off, ok to merge?

**Comment by locks** [comment]
@amyrlam Probably some hiccup. given the change I think it's fine to merge!

**Pull request merged**